main/src/library/MutList.flix

```diff
@@ -608,18 +603,16 @@ namespace MutList {
     /// Truncates the mutable list as needed.
     ///
```

Comment:
We should add documentation noting that there's a minimum capacity. Otherwise I'll be upset when I try to shrink my list to size 4 and it stays at size 8. :)